test/proto/benchmarks/control.proto

```diff
@@ -118,32 +67,45 @@ message ParetoParams {
   double alpha = 2;
 }
 
+message ClosedLoopParams {
+}
+
 message LoadParams {
   oneof load {
-    PoissonParams poisson = 1;
-    UniformParams uniform = 2;
-    DeterministicParams determ = 3;
-    ParetoParams pareto = 4;
+    ClosedLoopParams closed_loop = 1;
+    PoissonParams poisson = 2;
+    UniformParams uniform = 3;
+    DeterministicParams determ = 4;
+    ParetoParams pareto = 5;
   };
 }
 
+// presence of SecurityParams implies use of TLS
+message SecurityParams {
+  bool use_test_ca = 1;
```

Comment:
Is this useful? Do we want to test against a real prod server or something? In case we are going to test other security protocol in the future. Maybe let SecurityParams contain a sub message for tls or rename it to TlsSecurityParams. I am fine if this is done in another PR.